aiida_pseudo/cli/install.py

```diff
@@ -231,6 +231,23 @@ def cmd_install_pseudo_dojo(version, functional, relativistic, protocol, pseudo_
                 msg = f'md5 of pseudo for element {element} does not match that of the metadata {md5}'
                 echo.echo_critical(msg)
 
+        adjusted_cutoff_elements = []
```

Comment:
I wonder if we should explicit only do this logic for the PAW configurations? Such that we cannot accidentally start to apply this to other configurations that shouldn't have missing cutoffs? In any case, we should add a comment why this logic is here and what it is doing.

Reply:
I can do this only for PAW configurations and add a comment